editor/opening_hours_ui.hpp

```diff
@@ -0,0 +1,102 @@
+#pragma once
+
+#include "std/set.hpp"
+#include "std/vector.hpp"
+
+#include "3party/opening_hours/opening_hours.hpp"
+
+namespace editor
+{
+namespace ui
+{
+
+using TOpeningDays = set<osmoh::Weekday>;
```

Comment:
No, I'm not kidding. I've checked on many online and offline compilers, even on MSVC one. All of them are correctly give compilation errors (and always did it in the past).